pom.xml

```diff
@@ -100,10 +100,6 @@
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-storage-format</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.confluent</groupId>
-            <artifactId>kafka-connect-storage-hive</artifactId>
-        </dependency>
```

Comment:
Is the intention to remove this from the build but not remove the code? Why not just remove the module altogether?